include/unifex/task.hpp

```diff
@@ -632,18 +650,18 @@ struct _sr_thunk_task<T>::type final : coro_holder {
  * Await the given sa_task<> in a context that will deliver stop requests from
  * the receiver on the expected scheduler.
  */
-template <typename T>
+template <typename T, bool nothrow>
 typename _sr_thunk_task<T>::type
```

Comment:
Maybe this should just return a `_sr_thunk_task<T, nothrow>::type` right now instead of being a TODO.